daemon/daemon.go

```diff
@@ -413,19 +413,19 @@ func (d *Daemon) init() error {
 	fw.WriteString(common.FmtDefineArray("ROUTER_IP", d.conf.NodeAddress.IPv6Address))
 
 	ipv4GW := d.conf.NodeAddress.IPv4Address
-	fmt.Fprintf(fw, "#define IPV4_GATEWAY %#x\n", binary.LittleEndian.Uint32(ipv4GW))
+	fmt.Fprintf(fw, "#define IPV4_GATEWAY %#x\n", byteorder.NetworkToHostSliceU32(ipv4GW))
```

Comment:
I'm getting confused how all of this can work. IPV4_GATEWAY must be written in network byte order.